components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -19,7 +19,14 @@
         :data-qa="`${filter.label} checkbox`"
         class="mb-3"
       >
-        {{ filter.label }}
+        <span
```

Comment:
As this element is duplicated below, move it into its own component, components/generic/ColourSwatch.vue, accepting hex code and colour name as props.